tools/rna_tools/vienna_rna/rnaalifold.xml

```diff
@@ -112,7 +114,17 @@
                 <option value="0">0: Deactivates the calculation of the pair probabilities, saving about 50% in runtime</option>
                 <option value="1" >1: Calculate the partition function and base pairing probability matrix</option>				
             </param>
-            <param name="mea" type="float" value="1.0" label="Gamma Value" help="Calculate an MEA (maximum expected accuracy) structure. The parameter gamma tunes the importance of correctly predicted pairs versus unpaired bases. Thus, for small values of gamma the MEA structure will contain only pairs with very high probability. where the expected accuracy is computed from the pair probabilities: each base pair (i,j) gets a score 2*gamma*p_ij and the score of an unpaired base is given by the probability of not forming a pair." argument="--MEA"/>
+
+            <conditional name="measelect">
```

Comment:
mh, wait, we can simply make this float value optional, isn't it ... then we don't need the conditional ...